core-api/src/main/java/org/silverpeas/core/date/Temporal.java

```diff
@@ -38,7 +38,9 @@
  * interface with a generic type. Temporal can be then considered as a type generator that satisfies
  * a bound constraint (a fix point).
  * @param <T> A temporal type this class should generate.
+ * @deprecated Use the java.time API
  */
+@Deprecated
 public interface Temporal<T extends Temporal<? super T>> extends Cloneable {
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS1133)